<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSettingsDialog</class>
 <widget class="QDialog" name="CSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Preferences</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res/inyokaedit_resources.qrc">
    <normaloff>:/images/inyokaedit_64x64.png</normaloff>:/images/inyokaedit_64x64.png</iconset>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedKingdom"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="generalTab">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_3">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::ExpandingFieldsGrow</enum>
       </property>
       <item row="0" column="1">
        <widget class="QCheckBox" name="codeCompletionCheck">
         <property name="text">
          <string>Code completion</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QCheckBox" name="linkCheckingCheck">
         <property name="text">
          <string>Enable link checking</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QCheckBox" name="scrollbarSyncCheck">
         <property name="text">
          <string>Scrollbar synchronization</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QCheckBox" name="previewAlongsideCheck">
         <property name="text">
          <string>Preview alongside</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="previewInEditorCheck">
         <property name="text">
          <string>Preview in editor</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="autosaveLabel">
         <property name="text">
          <string>Autosave time (in sec)</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="reloadPreviewKeyLabel">
         <property name="text">
          <string>Reload preview key</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="timedPreviewLabel">
         <property name="text">
          <string>Auto reload preview (in sec)</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="numberRecentFilesLabel">
         <property name="text">
          <string>Number of entries in recent files list</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QSpinBox" name="autosaveEdit">
         <property name="toolTip">
          <string>Time which to autosave an article after (sec, 0 means disabled)</string>
         </property>
         <property name="maximum">
          <number>999</number>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="reloadPreviewKeyEdit">
         <property name="toolTip">
          <string>Select input box and press the desired reload key. Press DEL to remove the key.</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QSpinBox" name="timedPreviewsEdit"/>
       </item>
       <item row="8" column="1">
        <widget class="QSpinBox" name="numberRecentFilesEdit">
         <property name="value">
          <number>5</number>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="GuiLangLabel">
         <property name="text">
          <string>Gui language</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QComboBox" name="GuiLangCombo">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="fontTab">
      <attribute name="title">
       <string>Font &amp;&amp; Color</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_2">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::ExpandingFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="fontFamilyLabel">
         <property name="text">
          <string>Font</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QFontComboBox" name="fontComboBox"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="fontSizeLabel">
         <property name="text">
          <string>Font size</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QDoubleSpinBox" name="fontSizeEdit">
         <property name="decimals">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="Line" name="line2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="styleLabel">
         <property name="text">
          <string>Style</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="styleFilesBox"/>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QTableWidget" name="styleTable">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::NoSelection</enum>
         </property>
         <property name="rowCount">
          <number>14</number>
         </property>
         <property name="columnCount">
          <number>4</number>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>80</number>
         </attribute>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <row/>
         <column/>
         <column/>
         <column/>
         <column/>
         <item row="0" column="0">
          <property name="text">
           <string/>
          </property>
         </item>
         <item row="0" column="1">
          <property name="text">
           <string notr="true">---</string>
          </property>
          <property name="textAlignment">
           <set>AlignHCenter|AlignVCenter|AlignCenter</set>
          </property>
          <property name="flags">
           <set>NoItemFlags</set>
          </property>
         </item>
         <item row="0" column="2">
          <property name="text">
           <string notr="true">---</string>
          </property>
          <property name="textAlignment">
           <set>AlignHCenter|AlignVCenter|AlignCenter</set>
          </property>
          <property name="flags">
           <set>NoItemFlags</set>
          </property>
         </item>
         <item row="0" column="3">
          <property name="text">
           <string notr="true">---</string>
          </property>
          <property name="textAlignment">
           <set>AlignHCenter|AlignVCenter|AlignCenter</set>
          </property>
          <property name="flags">
           <set>NoItemFlags</set>
          </property>
         </item>
         <item row="1" column="0">
          <property name="text">
           <string/>
          </property>
         </item>
         <item row="1" column="1">
          <property name="text">
           <string notr="true">---</string>
          </property>
          <property name="textAlignment">
           <set>AlignHCenter|AlignVCenter|AlignCenter</set>
          </property>
          <property name="flags">
           <set>NoItemFlags</set>
          </property>
         </item>
         <item row="1" column="2">
          <property name="text">
           <string notr="true">---</string>
          </property>
          <property name="textAlignment">
           <set>AlignHCenter|AlignVCenter|AlignCenter</set>
          </property>
          <property name="flags">
           <set>NoItemFlags</set>
          </property>
         </item>
         <item row="1" column="3">
          <property name="text">
           <string notr="true">---</string>
          </property>
          <property name="textAlignment">
           <set>AlignHCenter|AlignVCenter|AlignCenter</set>
          </property>
          <property name="flags">
           <set>NoItemFlags</set>
          </property>
         </item>
         <item row="2" column="0">
          <property name="text">
           <string/>
          </property>
         </item>
         <item row="2" column="1">
          <property name="text">
           <string/>
          </property>
          <property name="checkState">
           <enum>Unchecked</enum>
          </property>
          <property name="flags">
           <set>ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
         <item row="2" column="2">
          <property name="text">
           <string/>
          </property>
          <property name="textAlignment">
           <set>AlignLeft|AlignVCenter</set>
          </property>
          <property name="checkState">
           <enum>Unchecked</enum>
          </property>
          <property name="flags">
           <set>ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
         <item row="2" column="3">
          <property name="text">
           <string/>
          </property>
         </item>
         <item row="3" column="0">
          <property name="text">
           <string/>
          </property>
         </item>
         <item row="3" column="1">
          <property name="text">
           <string/>
          </property>
          <property name="checkState">
           <enum>Unchecked</enum>
          </property>
          <property name="flags">
           <set>ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
         <item row="3" column="2">
          <property name="text">
           <string/>
          </property>
          <property name="checkState">
           <enum>Unchecked</enum>
          </property>
          <property name="flags">
           <set>ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
         <item row="3" column="3">
          <property name="text">
           <string/>
          </property>
         </item>
         <item row="4" column="0">
          <property name="text">
           <string/>
          </property>
         </item>
         <item row="4" column="1">
          <property name="text">
           <string/>
          </property>
          <property name="checkState">
           <enum>Unchecked</enum>
          </property>
          <property name="flags">
           <set>ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
         <item row="4" column="2">
          <property name="text">
           <string/>
          </property>
          <property name="checkState">
           <enum>Unchecked</enum>
          </property>
          <property name="flags">
           <set>ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
         <item row="4" column="3">
          <property name="text">
           <string/>
          </property>
         </item>
         <item row="5" column="0">
          <property name="text">
           <string/>
          </property>
         </item>
         <item row="5" column="1">
          <property name="text">
           <string/>
          </property>
          <property name="checkState">
           <enum>Unchecked</enum>
          </property>
          <property name="flags">
           <set>ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
         <item row="5" column="2">
          <property name="text">
           <string/>
          </property>
          <property name="checkState">
           <enum>Unchecked</enum>
          </property>
          <property name="flags">
           <set>ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
         <item row="5" column="3">
          <property name="text">
           <string/>
          </property>
         </item>
         <item row="6" column="0">
          <property name="text">
           <string/>
          </property>
         </item>
         <item row="6" column="1">
          <property name="text">
           <string/>
          </property>
          <property name="checkState">
           <enum>Unchecked</enum>
          </property>
          <property name="flags">
           <set>ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
         <item row="6" column="2">
          <property name="text">
           <string/>
          </property>
          <property name="checkState">
           <enum>Unchecked</enum>
          </property>
          <property name="flags">
           <set>ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
         <item row="6" column="3">
          <property name="text">
           <string/>
          </property>
         </item>
         <item row="7" column="0">
          <property name="text">
           <string/>
          </property>
         </item>
         <item row="7" column="1">
          <property name="text">
           <string/>
          </property>
          <property name="checkState">
           <enum>Unchecked</enum>
          </property>
          <property name="flags">
           <set>ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
         <item row="7" column="2">
          <property name="text">
           <string/>
          </property>
          <property name="checkState">
           <enum>Unchecked</enum>
          </property>
          <property name="flags">
           <set>ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
         <item row="7" column="3">
          <property name="text">
           <string/>
          </property>
         </item>
         <item row="8" column="0">
          <property name="text">
           <string/>
          </property>
         </item>
         <item row="8" column="1">
          <property name="text">
           <string/>
          </property>
          <property name="checkState">
           <enum>Unchecked</enum>
          </property>
          <property name="flags">
           <set>ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
         <item row="8" column="2">
          <property name="text">
           <string/>
          </property>
          <property name="checkState">
           <enum>Unchecked</enum>
          </property>
          <property name="flags">
           <set>ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
         <item row="8" column="3">
          <property name="text">
           <string/>
          </property>
         </item>
         <item row="9" column="0">
          <property name="text">
           <string/>
          </property>
         </item>
         <item row="9" column="1">
          <property name="text">
           <string/>
          </property>
          <property name="checkState">
           <enum>Unchecked</enum>
          </property>
          <property name="flags">
           <set>ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
         <item row="9" column="2">
          <property name="text">
           <string/>
          </property>
          <property name="checkState">
           <enum>Unchecked</enum>
          </property>
          <property name="flags">
           <set>ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
         <item row="9" column="3">
          <property name="text">
           <string/>
          </property>
         </item>
         <item row="10" column="0">
          <property name="text">
           <string/>
          </property>
         </item>
         <item row="10" column="1">
          <property name="text">
           <string/>
          </property>
          <property name="checkState">
           <enum>Unchecked</enum>
          </property>
          <property name="flags">
           <set>ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
         <item row="10" column="2">
          <property name="text">
           <string/>
          </property>
          <property name="checkState">
           <enum>Unchecked</enum>
          </property>
          <property name="flags">
           <set>ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
         <item row="10" column="3">
          <property name="text">
           <string/>
          </property>
         </item>
         <item row="11" column="0">
          <property name="text">
           <string/>
          </property>
         </item>
         <item row="11" column="1">
          <property name="text">
           <string/>
          </property>
          <property name="checkState">
           <enum>Unchecked</enum>
          </property>
          <property name="flags">
           <set>ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
         <item row="11" column="2">
          <property name="text">
           <string/>
          </property>
          <property name="checkState">
           <enum>Unchecked</enum>
          </property>
          <property name="flags">
           <set>ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
         <item row="11" column="3">
          <property name="text">
           <string/>
          </property>
         </item>
         <item row="12" column="0">
          <property name="text">
           <string/>
          </property>
         </item>
         <item row="12" column="1">
          <property name="text">
           <string/>
          </property>
          <property name="checkState">
           <enum>Unchecked</enum>
          </property>
          <property name="flags">
           <set>ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
         <item row="12" column="2">
          <property name="text">
           <string/>
          </property>
          <property name="checkState">
           <enum>Unchecked</enum>
          </property>
          <property name="flags">
           <set>ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
         <item row="12" column="3">
          <property name="text">
           <string/>
          </property>
         </item>
         <item row="13" column="0">
          <property name="text">
           <string/>
          </property>
         </item>
         <item row="13" column="1">
          <property name="text">
           <string/>
          </property>
          <property name="checkState">
           <enum>Unchecked</enum>
          </property>
          <property name="flags">
           <set>ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
         <item row="13" column="2">
          <property name="text">
           <string/>
          </property>
          <property name="checkState">
           <enum>Unchecked</enum>
          </property>
          <property name="flags">
           <set>ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
         <item row="13" column="3">
          <property name="text">
           <string/>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="connectionTab">
      <attribute name="title">
       <string>Connection</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="inyokaUrlLabel">
         <property name="text">
          <string>URL to Inyoka wiki</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="inyokaUrlEdit"/>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="articleImageDownloadCheck">
         <property name="text">
          <string>Download article images automatically</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="proxyHostNameLabel">
         <property name="text">
          <string>Proxy host name</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="proxyHostNameEdit"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="proxyPortLabel">
         <property name="text">
          <string>Port</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QSpinBox" name="proxyPortSpinBox">
         <property name="maximum">
          <number>99999</number>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="proxyUserNameLabel">
         <property name="text">
          <string>User name</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="proxyUserNameEdit"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="proxyPasswordLabel">
         <property name="text">
          <string>Password</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="proxyPasswordEdit">
         <property name="echoMode">
          <enum>QLineEdit::Normal</enum>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLabel" name="proxyPasswordHintLabel">
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:8pt; color:#ff0000;&quot;&gt;Attention: Password not encrypted!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0" colspan="2">
        <widget class="Line" name="line3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QCheckBox" name="WindowsUpdateCheck">
         <property name="text">
          <string>Check for updates (Windows only)</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pluginsTab">
      <attribute name="title">
       <string>Plugins</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_4">
       <item row="0" column="0" colspan="2">
        <widget class="QTableWidget" name="pluginsTable">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>1</verstretch>
          </sizepolicy>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::NoSelection</enum>
         </property>
         <property name="iconSize">
          <size>
           <width>80</width>
           <height>80</height>
          </size>
         </property>
         <property name="showGrid">
          <bool>false</bool>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res/inyokaedit_resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>228</x>
     <y>352</y>
    </hint>
    <hint type="destinationlabel">
     <x>228</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>228</x>
     <y>352</y>
    </hint>
    <hint type="destinationlabel">
     <x>228</x>
     <y>184</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
